src/chainparams.h

```diff
@@ -104,6 +104,8 @@ class CChainParams
     std::string GetFoundersRewardAddressAtIndex(int i) const;
     /** Enforce coinbase consensus rule in regtest mode */
     void SetRegTestCoinbaseMustBeProtected() { consensus.fCoinbaseMustBeProtected = true; }
+    /** Enable ZIP209 in regtest mode */
```

Comment:
Strictly speaking this function can be used to enable it in any mode.

Reply:
Good point. I've moved this into chainparams.cpp since we only need to call this in regtest mode.